src/index.js

```diff
@@ -34,6 +34,24 @@ function parsePart(part) {
   });
 }
 
+function validatePart(timeElementValues, validator){
+  let elements = [], isValid = true;
+
+  if(timeElementValues != null) {
+    timeElementValues.forEach((value)=> {
+      elements = elements.concat(Object.values(value))
+    });
+
+    isValid = elements.every((element)=> element >= validator.min && element <= validator.max );
+
+    if(!isValid){
+      throw new Error(
+        `Invalid time element. Range should be within [${validator.min} - ${validator.max}].`
```

Comment:
Do you have ideas on how we can specify which time range is invalid? (Is it the month part, the minute part, etc).

Reply:
Hi Yes, I ll push the new changes where you can find the name of the time element